README.rst

```diff
@@ -279,6 +279,25 @@ Install Python bindings
 
 .. installation-end-marker-do-not-remove
 
+
+Testing Dali build
+^^^^^^^^^^^^^^^^^^
+For the purpose of testing Dali, we maintain DALI_extra_ repository, where all testing data can be found.
+Should you want to run tests for Dali build, please obtain DALI_extra_ and put path to root repo folder
```

Comment:
How about:
```
Before running the tests `DALI_EXTRA_PATH` should be set and point to the content of DALI_extra_.
```
It would be good to mention which test can be run with this repo, also we have not adjusted all of our unit tests to utilize it. So we should wait until we do.